datasource/jaxb/src/main/resources/model.v1.xsd

```diff
@@ -226,4 +226,96 @@
         </complexType>
     </element>
 
+    <element name="dataSetMap">
```

Comment:
Can we use hyphens instead of camel-case for the attributes and elements here?

(We try and avoid using these for XML types.)

Reply:
Done.